### PR TITLE
magento/devdocs#: Add documentation about @magentoConfigFixture annotation

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds a documentation about `@magentoConfigFixture` annotation to [GraphQL functional testing](https://devdocs.magento.com/guides/v2.3/graphql/functional-testing.html) page.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/functional-testing.html

## Links to Magento source code

- [@magentoConfigFixture](https://github.com/magento/graphql-ce/pull/351)

CC: @keharper @VitaliyBoyko 

Thank you!

whatsnew
Added a description of the `@magentoConfigFixture` annotation to the [GraphQL functional testing](https://devdocs.magento.com/guides/v2.3/graphql/functional-testing.html) page.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/functional-testing.md [hunk 1]
```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
...
+    /**
+     * @magentoConfigFixture default_store checkout/options/enable_agreements 1
+     */
+    public function testGetActiveAgreement()
+    {
+        ...
+    }
+```
+
+In the background process, `@magentoConfigFixture` do the next before test execution:
```
Comment (rogyar) [inline guides/v2.3/graphql/functional-testing.md]: Please, adjust.
```suggestion
In the background process, `@magentoConfigFixture` performs the following action before test execution:  
```

File: guides/v2.3/graphql/functional-testing.md [hunk 2]
```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
...
+
+In the background process, `@magentoConfigFixture` do the next before test execution:  
+
+```sql
+INSERT INTO `core_config_data` (scope`, `scope_id`, `path`, `value`)
+VALUES
+	('stores', 1, 'checkout/options/enable_agreements', '1');
+```
+
+and automatically removes `checkout/options/enable_agreements` config key from database when test be completed.
```
Comment (rogyar) [inline guides/v2.3/graphql/functional-testing.md]: Please, adjust 
```suggestion
and automatically removes `checkout/options/enable_agreements` config key from the database after the test has been completed.
```

File: guides/v2.3/graphql/functional-testing.md [hunk 3]
```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
...
+#### Syntax
+
+```php
+/**
+ * @magentoConfigFixture <store_code>_store <config_key> <config_value>
+ */
+```
+
+where
+- `<store_code>` - Code of store. See `store`.`code` table
```
Comment (rogyar) [inline guides/v2.3/graphql/functional-testing.md]: I would make the description more transparent here in the following way for all arguments
```suggestion
- `<store_code>` - Store code. See the `store`.`code` database field value. 
```

Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: **fixed**

File: guides/v2.3/graphql/functional-testing.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: **fixed**

File: guides/v2.3/graphql/functional-testing.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: **fixed**

File: guides/v2.3/graphql/functional-testing.md (hunk 3 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: @rogyar , thank you for review!

All your suggestions have been implemented.

Thank you!

File: guides/v2.3/graphql/functional-testing.md [hunk 4]
```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
...
+
+In the background process, `@magentoConfigFixture` performs the following action before test execution:  
+
+```sql
+INSERT INTO `core_config_data` (scope`, `scope_id`, `path`, `value`)
+VALUES
+	('stores', 1, 'checkout/options/enable_agreements', '1');
+```
+
+and automatically removes `checkout/options/enable_agreements` config key from the database after the test has been completed.
```
Comment (keharper) [inline guides/v2.3/graphql/functional-testing.md]:
```suggestion
The fixture automatically removes the `checkout/options/enable_agreements` config key from the database after the test has been completed.
```

File: guides/v2.3/graphql/functional-testing.md [hunk 5]
```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
...
+    /**
+     * @magentoConfigFixture default_store checkout/options/enable_agreements 1
+     */
+    public function testGetActiveAgreement()
+    {
+        ...
+    }
+```
+
+In the background process, `@magentoConfigFixture` performs the following action before test execution:
```
Comment (keharper) [inline guides/v2.3/graphql/functional-testing.md]:
```suggestion
`@magentoConfigFixture` performs the following action  as a background process before test execution:  
```

File: guides/v2.3/graphql/functional-testing.md [hunk 6]
```diff
@@ -196,6 +196,50 @@ $registry->unregister('isSecureArea');
 $registry->register('isSecureArea', false);
 ```
 
+### Fixture configs
+
+Use `@magentoConfigFixture` annotation to set a custom config value. It supports a `store` scope only.
```
Comment (keharper) [inline guides/v2.3/graphql/functional-testing.md]:
```suggestion
Use the `@magentoConfigFixture` annotation to set a custom config value. It supports a `store` scope only.
```

Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: **fixed**

File: guides/v2.3/graphql/functional-testing.md (hunk 5 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: **fixed**

File: guides/v2.3/graphql/functional-testing.md (hunk 4 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: **fixed**

File: guides/v2.3/graphql/functional-testing.md (hunk 5 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/functional-testing.md]: @keharper , all your suggestions have been implemented.

Thank you!

**Comment by keharper** [comment]
@rogyar Please give your approval.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks @atwixfirster!

**Comment by atwixfirster** [comment]
> Thanks @atwixfirster!

With my pleasure, @keharper

**Pull request merged**